### PR TITLE
Fix slash repeat when you call current_path in capybara

### LINKED ISSUE #88: double slashes in routes when testing with capybara
Hi,

Since I've upgraded from Lotus 0.5.0 to Hanami 0.7.0 I'm getting the failure below for my feature tests:

```
Expected: "/customers/4/edit"
  Actual: "//customers/4/edit"
```

I haven't changed my tests, I have the same routes setup and under Lotus 0.5.0 I still don't get these failures.

This is the feature test I have:
```
it 'updates customer data' do
    create_action.call(create_params)
    create_action.customer.id.wont_be_nil
    visit "/customers/#{customer_id}/edit"
    current_path.must_equal("/customers/#{customer_id}/edit")

    fill_in('customer[vat_id]', with: var)
    click_button('Update customer')
    current_path.must_equal("/customers/#{customer_id}")
    assert page.has_content?(var), "The updated customer vat id should be #{var}."
  end
```
Here are my routes:

```
resources :projects
resources :persons
resources :languages
resources :customers
resources :services
get '/', to: 'home#index', as: :home
get '/configuration', to: 'configuration#index', as: :configuration
```

The current_path from the above block as printed in the terminal is
```
 //customers/38/edit.
```
When I go through the app manually and update some data, everything works. I'm going through the code if I've missed something when upgrading, but I can't find anything.

regards,
Seba

### PR DESCRIPTION
**Pull request by davydovanton**
Fixed #88

### CODE REVIEW

**Comment by davydovanton** [comment]
@hanami/core-team what do you think about this?

File: test/routing/http_router_test.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    def rack_request
+      TestRackRequest.new
+    end
+  end
+
+  describe '#rewrite_path_info' do
+    let(:env) { { 'SCRIPT_NAME' => '' } }
+    let(:request) { TestRequest.new }
+
+    it 'not repeat slashs' do
```
Comment (joneslee85) [inline test/routing/http_router_test.rb]: what do you think about `it 'rejects entries that are matching separator' do`?

**Comment by joneslee85** [comment]
besides the comment, it is :+1: 

cc @AlfonsoUceda

File: test/routing/http_router_test.rb (hunk 1 above)
Comment (davydovanton) [inline test/routing/http_router_test.rb]: Yes, it's better. Fixed :+1:

**Comment by jodosha** [comment]
It looks good to me folks. @sebastjan-hribar @davydovanton @joneslee85 @AlfonsoUceda Can you please also try this patch in an app and share the outcomes?

Thank you all! :green_heart:

**Comment by davydovanton** [comment]
@jodosha done:
https://asciinema.org/a/6q0nv98kt75ox5q72xe0qx7g7

**Comment by AlfonsoUceda** [comment]
@davydovanton it works perfect! thank you so much! cc @jodosha

**Comment by davydovanton** [comment]
You're welcome :)

**Comment by joneslee85** [comment]
Confirm that it works

**Comment by sebastjan-hribar** [comment]
I confirm the fix works. Thank you!

regards,
seba

**Comment by jodosha** [comment]
Thank you all! :100:

**Pull request merged**